POEApi.Model/ProxyMapper.cs

```diff
@@ -298,5 +299,18 @@ internal static List<Tab> GetTabs(List<JSONProxy.Tab> tabs)
                 throw;
             }
         }
+
+        public static ChargeInfo GetCharges(List<JSONProxy.Property> list)
```

Comment:
I will put a try catch around the creation of the ChargeInfo object. I will return a new charge(1,1) if it fails.